src/beautifiers/beautysh.coffee

```diff
@@ -0,0 +1,14 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class BashBeautify extends Beautifier
+  name: "beautysh"
+  link: "https://github.com/bemeurer/beautysh"
+  options: {
+    Bash: true
+  }
+
+  beautify: (text, language, options) ->
+    file = @tempFile("input", text)
+    @run('beautysh', [ file ], help: { link: "https://github.com/bemeurer/beautysh" })
```

Comment:
`beautysh` will need to be installed for Travis CI.

Add `pip install beautysh` to https://github.com/Glavin001/atom-beautify/blob/master/.travis.yml#L75